public/static/docs/command-reference/update.md

```diff
@@ -69,3 +77,43 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
```

Comment:
feels that we now have two similar examples - let's keep one of them and make a link to another?

Reply:
I am not sure how to do that, as the flow of two examples are different (one starts with --rev, other does not mention it at all).

Reply:
agreed.

Reply:
It's baiscaly the same as **Example: Fixed revisions and updating to different revision** in `dvc import` though. Maybe we should just copy/paste it or use similar terminology i.e. "fixed revision".